crates/stages/src/stages/bodies.rs

```diff
@@ -171,6 +172,7 @@ impl<DB: Database, D: BodyDownloader, C: Consensus> Stage<DB> for BodyStage<D, C
         // - We reached our target and the target was not limited by the batch size of the stage
         let capped = target < previous_stage_progress;
         let done = highest_block < target || !capped;
+        info!(target : "sync::stages::bodies", stage_progress = highest_block, done = done, "Finished stage.");
```

Comment:
not entirely, should maybe say "synced until" or smth since it will sync over multiple runs

Reply:
changed to `Sync iteration finished`